### ISSUE TITLE
Fails to run on macOS / ARM

### ISSUE THREAD

**Issue by certik** [conda-forge/fpm-feedstock#7]
The fpm version 0.4.0 works on macOS / Intel. But on macOS / ARM I get:
```console
$ conda create -n fpm2 fpm=0.4.0
$ conda activate fpm2
$ fpm
zsh: killed     fpm
```

**Comment by awvwgk** [comment]
@certik can you check whether this is still an issue?

**Comment by certik** [comment]
I tried:
```console
~ $ mamba create -n fpm3 fpm=0.5.0
~ $ conda activate fpm3
(fpm3) ~ $ which fpm
/Users/certik/miniforge3/envs/fpm3/bin/fpm
(fpm3) ~ $ fpm
Fortran Package Manager:
 
USAGE: fpm [ SUBCOMMAND [SUBCOMMAND_OPTIONS] ]|[--list|--help|--version]
       where SUBCOMMAND is commonly new|build|run|test

 subcommand may be one of

  build     Compile the package placing results in the "build" directory
  help      Display help
  list      Display this list of subcommand descriptions
  new       Create a new Fortran package directory with sample files
  run       Run the local package application programs
  test      Run the test programs
  update    Update and manage project dependencies
  install   Install project

 Enter "fpm --list" for a brief list of subcommand options. Enter
 "fpm --help" or "fpm SUBCOMMAND --help" for detailed descriptions.


fpm: Leaving directory '/Users'
(fpm3) ~ $ fpm --version
Version:     0.5.0, alpha
Program:     fpm(1)
Description: A Fortran package manager and build system
Home Page:   https://github.com/fortran-lang/fpm
License:     MIT
OS Type:     macOS
```
So I think it works!

Do you know what exactly the problem was and what fixed it?

**Comment by awvwgk** [comment]
No idea, but it is fixed now ;).

**Comment by certik** [comment]
I think it was solely fixed by #8, because that's the only PR merged. The PR merged right before #8 is #6, and that one had the bug above.

In #8, it seems the main relevant change is in https://github.com/conda-forge/fpm-feedstock/pull/8/files#diff-d7075654874cb08007a21aaab3ecd4b3453a9087e7505d034d548b8938b599bc, so that probably fixed it somehow.
